region/crateria/central/Parlor and Alcatraz.json

```
@@ -1122,24 +1146,98 @@
-        "canPreciseCeilingClip",
-        "canTrickyUseFrozenEnemies",
+        "h_ZebesIsAwake",
+        "canBePatient",
         {"or": [
-          "Morph",
-          "canUseGrapple"
+          "canBeVeryPatient",
+          {"ammo": {"type": "Super", "count": 1}}
         ]},
+        "canPreciseCeilingClip",
+        "canTrickyUseFrozenEnemies",
+        "canUseGrapple",
         "canXRayStandUp"
       ],
       "note": [
-        "Wait for a global geemer to make the long trip along the map.",
+        "Wait for a global Geemer to make the long trip along the map, or shoot a Super 20 to 30 seconds after entering the room to knock it off the ceiling and save a lot of time.",
         "Freeze it just after it starts climbing the bomb blocks.",
-        "Then use Xray to standup before jumping through the ceiling.",
-        "A second Gemmer can be killed using Grapple to enable use of Xray, if Morph is not available."
+        "Spin-jump above it, which will put Samus into a forced crouch where X-Ray cannot be used.",
+        "Use Grapple to kill a a second Geemer, restoring the ability to use X-Ray.",
+        "Use X-Ray to stand up, then jump up through the ceiling."
       ],
-      "devNote": "FIXME: An RMode forced standup can be used for this strat but it requires reserves > etanks to farm the ripper, or a GMode Crystal Flash plus a way to damage down again."
+      "devNote": [
+        "Morph could also be used instead of Grapple, but with Morph there is an easier strat that goes up through the tunnel to the left."
+      ]
+    },
+    {
+      "link": [5, 8],
+      "name": "HiJump Ceiling Clip Alcatraz Escape",
+      "notable": true,
+      "requires": [
+        "HiJump",
+        "h_ZebesIsAwake",
+        {"or": [
+          "canBeVeryPatient",
+          {"ammo": {"type": "Super", "count": 1}}
+        ]},
+        "canPreciseCeilingClip",
+        "canTrickyUseFrozenEnemies",
+        {"enemyDamage": {
+          "enemy": "Geemer (blue)",
+          "type": "contact",
+          "hits": 1
+        }},
+        "canInsaneJump"
+      ],
+      "note": [
+        "Wait for a global Geemer to make the long trip along the map, or shoot a Super 20 to 30 seconds after entering the room to knock it off the ceiling and save a lot of time.",
+        "Hold right against the bomb blocks, take damage from the Geemer, jump, aim-down, and shoot the Geemer shortly before landing.",
+        "Samus should end up standing on the ground with the Geemer frozen on the wall slightly above the ground (overlapping Samus' hitbox).",
+        "Facing the bomb blocks and pressed against them, jump while holding down (but not crouched) and then press forward at a precise time to break the aim-down pose.",
+        "If successful, Samus will clip up through the ceiling.",
+        "This can be attempted several times before the Geemer thaws."
+      ]
+    },
+    {
+      "link": [5, 8],
+      "name": "G-Mode Forced Standup Alcatraz Escape",
```

```suggestion
      "name": "Alcatraz Escape G-Mode Crystal Flash Then Forced Standup",
```
Might be better with a bit more detail in the name